docs/en/observability/monitor-servers.asciidoc

```diff
@@ -32,3 +32,115 @@ This metric relies on the same indices as the logs.
 
 For information about which required fields the {metrics-app} uses to display host metrics, see the
 <<metrics-app-fields,Metrics field reference>>.
+
+[[enhanced-host-metrics]]
+== Enhanced host metrics
+
+Without leaving the <<view-infrastructure-metrics,*Inventory*>> page, you can view enhanced metrics relating to each host
+running in your infrastructure. On the waffle map, select the host to display the host metrics
+overlay.
+
+The enhanced metrics overlay contains the following tabs:
+
+// This is collapsed by default
+[%collapsible]
+.*Metrics*
+====
+
+[role="screenshot"]
+image::images/metrics-overlay.png[Host metrics]
+
+The *Metrics* tab displays CPU, load, memory, and network metrics relating to the host.
+You can change the time range to view metrics over the last 15 minutes, hour, 3 hours, 24 hours, or
+over the previous seven days. You can also hover over a specific time period on a chart to compare the various metrics
+at that given time.
+
+|=== 
+
+| *CPU* | Averages of `system.cpu.user.pct` and `system.cpu.system.pct`.
```

Comment:
This should be Averages of `system.cpu.user.pct / system.cpu.cores` and `system.cpu.system.pct / system.cpu.cores`

Reply:
Thanks @phillipb I made the update.